fairscale/experimental/nn/ssd_offload.py

```diff
@@ -237,61 +245,129 @@ def unwrap(e: Any) -> torch.Tensor:
                 e.to_file()
         return r
 
+    @classmethod
+    def __unpickle__(
+        cls: Type[SsdTensorHandle], shape: torch.Size, dtype: torch.dtype, requires_grad: bool, filename: str
+    ) -> "SsdTensorHandle":
+        result = cls(shape, dtype, requires_grad)
+        result.point_to_file(filename, 0)
+        result._unpickle_f = io.open(result.filename, "wb")
+        return result
+
+    def __reduce_ex__(self, proto: int) -> Tuple[Any, Any, Any, Any]:
+        byte_iter = None
+        filename = self.filename
+        if self.override_directory_path is not None:
+            head, tail = os.path.split(self.filename)
+            filename = os.path.join(self.override_directory_path, tail)
+            print(f"overridding path: {os.path.join(head, tail)} to {filename}")
```

Comment:
should this be a logging debug statement instead of print?

Reply:
I will remove the print. That was left in accidentally when I was implementing the feature.